### PR TITLE
Issue 66: Return block identifier in account call.

### PR DESCRIPTION
**Pull request by jgimeno**

### CODE REVIEW

File: cosmos/launchpad/account.go [hunk 1]
```diff
@@ -15,7 +19,23 @@ func (l Launchpad) AccountBalance(ctx context.Context, request *types.AccountBal
 		return nil, ErrNodeConnection
 	}
 
+	height, err := strconv.ParseInt(resp.Height, 10, 64)
```
Comment (ilgooz) [inline cosmos/launchpad/account.go:22]: Can you please change the corresponding `swagger.yml` prop to `integer` instead and regenerate the client?

File: cosmos/launchpad/account_test.go [hunk 2]
```diff
@@ -4,6 +4,11 @@ import (
 	"context"
 	"testing"
 
+	"github.com/antihax/optional"
+	openapi "github.com/tendermint/cosmos-rosetta-gateway/cosmos/launchpad/client/tendermint/generated"
+
```
Comment (ilgooz) [inline cosmos/launchpad/account_test.go:9]:
```suggestion
```

File: cosmos/launchpad/account_test.go [hunk 3]
```diff
@@ -4,6 +4,11 @@ import (
...
+	mocks2 "github.com/tendermint/cosmos-rosetta-gateway/cosmos/launchpad/client/tendermint/mocks"
+
```
Comment (ilgooz) [inline cosmos/launchpad/account_test.go:11]:
```suggestion
```

and gofmt after to reorder things?

**Review by ilgooz — CHANGES_REQUESTED** [review]
LGMT! Just a few tweaks.

File: cosmos/launchpad/account.go (hunk 1 above)
Comment (ilgooz) [reply cosmos/launchpad/account.go]: done by 96ca7e1

**Review by ilgooz — APPROVED** [review]
Tested!

**Review by sahith-narahari — APPROVED** [review]

**Pull request merged**